test/classification/training/loggers/test_checkpointer.py

```diff
@@ -21,57 +21,65 @@ def test_checkpointer(self) -> None:
             **log_manager_config,
             checkpoint_dir=self.test_dir,
             checkpoint_runway=3,
-            checkpoint_metric="f1:max",
+            checkpoint_metric="task/dataset/valid/f1:max",
```

Comment:
Ok so what's going on with these? Not actually a valid metric specification? And with the updated tests, we'd catch this now?

Reply:
right—`f1:max` is valid if passed to the checkpointer directly.
however, in any real application, `checkpoint_metric` is passed through the `trainer`, which formats the metric name with the applicable task/dataset information. 
that's where the silent bug was happening.